packages/docusaurus-mdx-loader/src/remark/transformImage/index.js

```diff
@@ -18,21 +18,22 @@ const {
 } = getFileLoaderUtils();
 
 const createJSX = (node, pathUrl) => {
-  node.type = 'jsx';
-  node.value = `<img ${node.alt ? `alt={"${escapeHtml(node.alt)}"} ` : ''}${
+  const jsxNode = node;
```

Comment:
why not rename the fn param directly?

Reply:
`no-param-reassign` rule - https://eslint.org/docs/rules/no-param-reassign